features/upgrade/logging/upgrade.feature

```diff
@@ -0,0 +1,130 @@
+Feature: Logging upgrading related features
+
+  # @author qitang@redhat.com
+  @admin
+  @destructive
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  Scenario: Cluster logging checking during cluster upgrade - prepare 
+    # prepare data
+    Given I switch to the first user
+    When I run the :new_project client command with:
+      | project_name | logging-upgrade-data-check |
+    Then the step should succeed
+    When I run the :new_app client command with:
+      | file | <%= BushSlicer::HOME %>/testdata/logging/loggen/container_json_log_template.json |
+    Then the step should succeed
+
+    # deploy clusterlogging, enable pvc for ES
+    Given logging operators are installed successfully
+    And default storageclass is stored in the :default_sc clipboard
+    When I process and create:
+      | f | <%= BushSlicer::HOME %>/testdata/logging/clusterlogging/clusterlogging-storage-template.yaml |
```

Comment:
As we only run once cases during upgrade. we should prepare a ES clusters. Let's use nodecount=3.

Reply:
Updated the template and the table, added a parameter `ES_NODE_COUNT`, set default value to `1`.